scripts/partition_manager_report.py

```diff
@@ -41,13 +41,23 @@ def print_region(region, size, pm_config):
     sorted_pm_config = sorted(sorted_pm_config, key=lambda x: pm_config[x]['address'])
 
     # Create text lines
-    lines = ['{}{}: {} ({}){}'.format(
+    lines = []
+    endspan = 0
+    for name in sorted_pm_config:
```

Comment:
Using an enumerator here, together with the later suggestion, will make code easier to re, imho.

```suggestion
    for i, name in enumerate(sorted_pm_config):
```